tls/s2n_fingerprint_ja3.c

```diff
@@ -204,5 +221,7 @@ struct s2n_fingerprint_method ja3_fingerprint = {
     /* The hash doesn't have to be cryptographically secure,
      * so the weakness of MD5 shouldn't be a problem. */
     .hash = S2N_HASH_MD5,
+    /* The hash string is a single MD5 digest represented as hex */
+    .hash_str_size = MD5_DIGEST_LENGTH * S2N_HEX_PER_BYTE,
```

Comment:
https://github.com/aws/s2n-tls/pull/4578#discussion_r1626594801 So I now remember why I had the hash size on the struct instead of the hash itself :) But two separate fields works just fine.